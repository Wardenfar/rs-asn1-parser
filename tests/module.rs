use asn1_parser::parse_asn1_file;

#[test]
fn empty_ok() {
    let m = parse_asn1_file(include_str!("../resources/empty-ok.asn1")).unwrap();
    insta::assert_json_snapshot!(m);
}

#[test]
fn garbage_err() {
    let e = parse_asn1_file(include_str!("../resources/garbage-err.asn1")).unwrap_err();
    insta::assert_display_snapshot!(e)
}
