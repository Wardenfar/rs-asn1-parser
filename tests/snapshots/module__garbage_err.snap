---
source: tests/module.rs
assertion_line: 12
expression: e

---
Parsing Error
